crates/globset/src/serde_impl.rs

```diff
@@ -1,5 +1,6 @@
-use serde::de::Error;
+use serde::de::{Error, Visitor};
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::fmt;
```

Comment:
Please group imports. std imports should come first, then an empty line, then third party imports (like `serde`) and then finally crate local imports.

Reply:
Or just write `std::fmt::Formatter` and `std::fmt::Result` below.

Reply:
Like so (see recent commit)?